demo/demo.html

```diff
@@ -0,0 +1,316 @@
+<main>
```

Comment:
I don't see this file being imported anywhere. Let's delete it if we don't use it.